circle.yml

```diff
@@ -158,6 +173,17 @@ flake8: &flake8
     - checkout
     - run: /opt/conda/bin/flake8 --statistics
 
+build_docs: &build_docs
+  working_directory: ~/conda
+  docker:
+    - image: readthedocs/build
+  steps:
+    - checkout
+    - run:
+        command: |
+          pip install -r utils/requirements-docs.txt
+          cd docs
+          make html
```

Comment:
If I'm reading this correctly, you're building the docs, but you're not deploying them anywhere.  Is that correct?  Should we figure out a way to push them somewhere, or perhaps trigger the temporary addition of a branch to RTD, followed by automated removal of that branch once the PR has been merged?

Reply:
Correct. I looked into doing that but RTD doesn't have a public API that allows that currently. Another option could be to expose these docs on the github pages branch?
The current state of the PR will at least try to remove the situation where the docs don't build at all.

Reply:
Yeah, maybe we can bring the github pages view back, and have that be our testbed.  With more than one docs PR at a time, it might get hairy, but it's better than nothing.